content/applications/sales/point_of_sale/configuration/epos_ssc.rst

```diff
@@ -72,9 +61,19 @@ it into your browser.
 
          .. tab:: Generate a self-signed certificate
 
-            After forcing the connection, sign in using your printer credentials to access the ePOS
-            printer settings. To sign in, enter `epson` in the :guilabel:`ID` field and your printer
-            serial number in the :guilabel:`Password` field.
+            Navigate to the ePOS' IP address (e.g., `https://192.168.1.25`) and force the
+            connection by clicking :guilabel:`Advanced` and :guilabel:`Proceed to [IP address]
+            (unsafe)`.
+
+            .. figure:: epos_ssc/browser-https-insecure.png
+               :scale: 75%
+               :alt: warning page about the connection privacy on Google Chrome
+
+               Warning page on Google Chrome, Windows 10
+
+            Then, sign in using your printer credentials to access the ePOS printer settings. To
+            sign in, enter `epson` in the :guilabel:`ID` field and your printer serial number in the
+            :guilabel:`Password` field.
```

Comment:
It's perfectly fine to me as is. The only thing I noticed is that in the other tabs, the instructions are divided into steps and numbered. Perhaps it would be good to do the same in this tab for consistency's sake, eg.:
1. Navigate to the ePOS' IP address (e.g., `https://192.168.1.25`) and force the connection by clicking :guilabel:`Advanced` and :guilabel:`Proceed to [IP address] (unsafe)`.
2. Sign in using your printer credentials to access the ePOS printer settings as follows: .....

Up to you :) Other than that, good to go for me!

Reply:
I'm adding this comment to another task since this change wouldn't fit the purpose of this PR